corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/collections.js

```diff
@@ -75,7 +85,7 @@ hqDefine("cloudcare/js/formplayer/menus/collections", function () {
             } else if (response.tree) {
                 // form entry time, doggy
                 _.extend(this, _.pick(response, this.formProperties));
-                FormplayerFrontend.trigger('startForm', response, this.app_id);
+                FormplayerFrontend.trigger('startForm', response, this.appId);
```

Comment:
This is unrelated to the bugfix.

This was referencing a non-existent var so it was always passing `undefined`. It doesn't seem to be used at all in the `startForm` handler so could potentially be removed.

Reply:
```suggestion
                FormplayerFrontend.trigger('startForm', response);
```